custom_code/management/commands/ingest_tns.py

```diff
@@ -51,6 +55,14 @@ def add_arguments(self, parser):
 
     def handle(self, lookback_days_nle=7., lookback_days_obs=3., **kwargs):
 
+
+        ## THIS IS A TEST, THE LINE SHOULD THROW AN ERROR AND SHOULD BE REMOVED!
+        vet_or_post_error(Target(
+            name="2025abc",
+            ra=10,
+            dec=10
+        ))
+        
```

Comment:
Should this be deleted?

Reply:
Haha yes! I even left a comment to make sure either I caught it or you did while reviewing :laughing: